### PR TITLE
amam/partner_homepage

### PR DESCRIPTION
**Pull request by mustofa-binary**
# Description

add homepage to partner programmes

Changes:
-   new route to pratner
-   change nav, footer, off-canvas menu navigations

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/partners/_partnership-opportunities.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import { Header, Text, CardStyle } from 'components/elements'
+import { localize, LocalizedLink } from 'components/localization'
+import { Container, SectionContainer, Flex } from 'components/containers'
+// import device from 'themes/device'
+// Icons
+import PartnerAffiliate from 'images/svg/partner-affiliate.svg'
+import PartnerPaymentAgent from 'images/svg/partner-payment-agent.svg'
+
+const StyledSection = styled(SectionContainer)``
+const StyledFlex = styled(Flex)``
```
Comment (arash-fs) [inline src/pages/partners/_partnership-opportunities.js]: what is the purpose of using `StyledSection` and `StyledFlex` when we didn't add any css? we can use `SectionContainer` and `Flex` instead.

Comment (mustofa-binary) [reply src/pages/partners/_partnership-opportunities.js]: thanks, let me remove them

File: src/pages/partners/_partnership-opportunities.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                ? css`
+                      svg > g > path:first-child {
+                          fill: var(--color-red);
+                      }
+                  `
+                : css`
+                      svg > g > g > path {
+                          fill: var(--color-red);
+                      }
+                      svg > g > g > g > path:last-child {
```
Comment (arash-fs) [inline src/pages/partners/_partnership-opportunities.js]: :smile: :+1:

Comment (mustofa-binary) [reply src/pages/partners/_partnership-opportunities.js]: the svg structure, is so fucked up here. both of the svg has different structure, i dont know any other way to do it 🙏

**Pull request merged**